<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAllFewestLineMapDlg</class>
 <widget class="QDialog" name="CAllFewestLineMapDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>212</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create All / Fewest Line Map</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="c_all_line">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>All-Line Map</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="c_fewest_line_subsets">
       <property name="toolTip">
        <string>Algorithm by Alan Penn and Bill Hillier</string>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Fewest-Line Map (Subsets)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="c_fewest_line_minimal">
       <property name="toolTip">
        <string>Algorithm by Alasdair Turner</string>
       </property>
       <property name="text">
        <string>Fewest-Line Map (Minimal)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>58</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="c_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>c_ok</sender>
   <signal>clicked()</signal>
   <receiver>CAllFewestLineMapDlg</receiver>
   <slot>OnOK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>c_cancel</sender>
   <signal>clicked()</signal>
   <receiver>CAllFewestLineMapDlg</receiver>
   <slot>OnCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnOK()</slot>
  <slot>OnCancel()</slot>
 </slots>
</ui>
